android/libraries/rib-base/src/main/java/com/badoo/ribs/android/ActivityStarter.kt

```diff
@@ -13,29 +13,30 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.uber.rib.core;
+package com.badoo.ribs.android
```

Comment:
We have `com.badoo.ribs.android.ActivityStarter.kt` and `com.uber.rib.core.ActivityStarter.java` is it expected or do we want to remove java version?

Reply:
I'd like to remove ribs-android package completely, but probably not in this PR.

Reply:
sure, i just pointed what I saw.